bincommon/commonflags.go

```diff
@@ -170,6 +176,18 @@ func TLSInsecure() bool {
 	return TLSInsecure
 }
 
+// ConnectionReuseRangeValidator returns a validator function that checks if the connection reuse range is valid
+// and set in httpOpts.
+func ConnectionReuseRangeValidator(httpOpts *fhttp.HTTPOptions) func(string) error {
+	return func(value string) error {
+		if err := httpOpts.ValidateConnectionReuseRange(value); err != nil {
+			return fmt.Errorf("fail to parse connection reuse range, err: %v", err)
```

Comment:
%w when wrapping errors - or maybe just return err as is?

Reply:
Updated